src/HoJBot.jl

```diff
@@ -24,29 +24,27 @@ import StructTypes
 
 const COMMAND_PREFIX = get(ENV, "HOJBOT_COMMAND_PREFIX", ",")
 
+function loadplugins()
+    includejlfiles("type/")
+    includejlfiles("command/")
+    includejlfiles("handler/")
+end
+
+function includejlfiles(dir)
+    wd = walkdir(joinpath(@__DIR__, dir))
+    commands = first(wd)
+    close(wd)
```

Comment:
Since it only works with single level, it's simpler to use `readdir` with `join = true` instead.

Reply:
but `readdir` is single level only aswell. Am I misunderstanding it?

Reply:
Right, that’s why I suggested that. The code would be simpler since you don’t need to deal with the tuple from `walkdir`.

Reply:
but on the other hand I only need the files (not the directories) and walkdir already sorts by type of element. `readdir` returns files and directories arbitrarily mixed.